v_rising/v_rising_vanilla/egg-v-rising.json

```
@@ -53,17 +63,27 @@
         {
             "name": "Game Settings Preset",
             "description": "What preset of game settings the server should run. Some settings may not apply after the save file is first created. Leave empty\/null if you are using a single-player uploaded save or a custom ServerGameSettings.json file in the `~\/save-data\/Settings\/` directory.",
-            "env_variable": "GAME_SETTINGS_PRESET",
+            "env_variable": "VR_PRESET",
             "default_value": "",
             "user_viewable": true,
             "user_editable": true,
-            "rules": "nullable|string|in:,DuoPvP,HardcorePvP,Level30PvE,Level30PvP,Level50PvE,Level50PvP,Level70PvE,Level70PvP,SoloPvP,StandardPvE_Easy,StandardPvE_Hard,StandardPvE,StandardPvP_Easy,StandardPvP_Hard,StandardPVP",
+            "rules": "nullable|string|in:,DuoPvP,DuoPvP_DailySiege,DuoPvP_NoSiege,DuoPvP_WeekendSiege,HardcoreDuoPvP,HardcorePvP,HardcoreTrioPvP,Level30PvE,Level30PvP,Level40PvE,Level40PvP,Level50PvE,Level50PvP,Level60PvE,Level60PvP,Level70PvE,Level70PvP,Level80PvE,Level80PvP,Level90PvE,Level90PvP,SoloPvP,StandardPvE,StandardPvP,StandardPvP_DailySiege,StandardPvP_NoSiege,StandardPvP_WeekendSiege,TrioPvP,TrioPvP_DailySiege,TrioPvP_NoSiege,TrioPvP_WeekendSiege",
+            "field_type": "text"
+        },
+        {
+            "name": "Game Difficulty Preset",
+            "description": "What preset of game difficulty the server should run. Some settings may not apply after the save file is first created.",
+            "env_variable": "VR_DIFFICULTY_PRESET",
+            "default_value": "Difficulty_Normal",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|in:Difficulty_Easy,Difficulty_Normal,Difficulty_Brutal",
```

I believe this needs to also allow an empty value (like Game Settings Preset) so some settings that can be put in `ServerGameSettings.json` are not overridden.

Some examples would be the `DropTableModifier_General` and `DurabilityDrainModifier` settings, server owners might want to change these but they are overridden in all three difficulty presets.

```suggestion
            "rules": "required|string|in:,Difficulty_Easy,Difficulty_Normal,Difficulty_Brutal",
```

---

It would also need to remove the required as even with an empty , in the allowed set that will fail

---

Ahh, good catch, looking into this now.

---

@sqamsqam Should be fixed now.